PostFreeSurfer/PostFreeSurferPipeline.sh

```diff
@@ -71,6 +71,30 @@ opts_AddOptional '--processing-mode' 'ProcessingMode' 'HCPStyleData|LegacyStyleD
 opts_AddOptional '--structural-qc' 'QCMode' 'yes|no|only' "whether to run structural QC, default 'yes'" 'yes'
 opts_AddOptional '--use-ind-mean' 'UseIndMean' 'YES or NO' "whether to use the mean of the subject's myelin map as reference map's myelin map mean, defaults to 'YES'" 'YES'
 
+#Longitudinal_mode modes:
+#NONE: cross-sectional processing (default)
+#TIMEPOINT_STAGE[1|2]: timepoint processing stage 1 or 2
+#TEMPLATE: template processing (must be run after all timepoints)
+
+#There are some specific conventions on timepoint and template processing directories:
+#In cross-sectional mode, 'Subject' label and subject study (timepoint) labels are treated as being the same by HCP in the original design. 
+#In longitudinal mode, 'Timepoint' is a study within a subject and, since there are multiple timepoints (studies) per subject, 
+#they must be labeled differently.
+#Timepoint label may be arbitrary but conventionally, should contain subject as part of name. E.g. if for subject 
+#HCA6002236 there are two timepoints, thay may be labeled HCA6002236_V1 and HCA6002236_V2. 
+#For crossectional (initial) processing, these are supplied instead of 'Subject' label to PreFreesurferPipeline. 
+#For the FreesurferPipeline-long, these are also supplied as timepoint labels, as well as chosen template label, e.g. HCA6002236_V1_V2. 
+#Then the same are supplied to PostFreesurferPipelineLongPrep and PostFreesurferPipeline in longitudinal mode.
+#internally, longitudinal timepoint directories will be named as: <Timepoint>.long.<Template>
+#Longitudinal template directory is named <Subject>.long.<Template>. 
+#Longitudinal Freesurfer files for timepoints are stored under <Timepoint>.long.<Template>/T1w/<Timepoint>.long.<Temlate>. 
+#Longitudinal Freesurfer files for template are stored under <Subject>.long.<Template>/T1w/<Template>. 
+
+opts_AddOptional '--longitudinal_mode' 'LongitudinalMode' 'NONE|TIMEPOINT_STAGE1|TIMEPOINT_STAGE2|TEMPLATE' "longitudinal processing mode" "NONE"
+opts_AddOptional '--longitudinal_template' 'LongitudinalTemplate' 'FS longitudial template label' "Longitudinal tetmplate if LongitudinalMode!=NONE" ""
+opts_AddOptional '--longitudinal_timepoint_list' 'LongitudinalTimepointList' 'FS longitudial timepoint list' "Longitudinal timepoint list '@' separated, if LongitudinalMode==TEMPLATE" ""
+opts_AddOptional '--longitudinal_timepoint' 'LongitudinalTimepoint' 'FS longitudinal timepoint label' "Longitudinal timepoint if LongitudinalMode==TIMEPOINT_STAGEX" ""
```

Comment:
If the default is the empty string, it doesn't need to be explicitly specified.  Please use dashes, not underscores, in the argument flags, to match the conventions we use.

Reply:
Done.

Reply:
I suggest committing edits and pushing to the branch, rather than marking things as resolved, so that the reviewers can verify that the edits are satisfactory.  I tend to think of it as the reviewer's responsibility to mark their comments as resolved.